internal/store/updatestorehandler.go

```diff
@@ -44,6 +49,10 @@ func (e *UpdateStoreHandler) Update(ctx context.Context, evt event.UpdateEvent,
 		e.log.Error(err, "error updating object in update", "object", evt.ObjectNew)
 		return
 	}
+	if err := e.driver.updateIngressStatuses(ctx, e.client); err != nil {
```

Comment:
@alex-bezek, we have a debouncer on the sync right? rather than passing the client through here, what do you think if we called `driver.Sync` after creates/updates/deletes?

Reply:
yeah, the debouncer should keep it from overwhelming things and will batch changes. We talked about actually just calling sync on every one of these handlers in here https://github.com/ngrok/kubernetes-ingress-controller/issues/149#issuecomment-1460670311 
I'm not sure if we still have the bug where creating the ingress before the moduleset doesn't re-trigger a sync, but I think we were thinking about how to solve that at the time.

Reply:
I was originally calling `driver.Sync`, but figured it might be better to scope it down to do less work. Happy to change it back if that would be better!